OsmAnd/src/net/osmand/aidl/OsmandAidlApi.java

```diff
@@ -1703,6 +1703,7 @@ public void registerNavDrawerItems(final Activity activity, ContextMenuAdapter a
 			String appPackage = entry.getKey();
 			for (NavDrawerItem item : entry.getValue()) {
 				Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(item.uri));
+				boolean isCopy = false;
```

Comment:
api should not limit items to be displayed even with same names. you fixed result, but not source of the issue.
looks like something wrong on telegram side.